modules/assignment/src/main/java/com/intuit/wasabi/assignment/cache/impl/NoopAssignmentsMetadataCacheImpl.java

```diff
@@ -0,0 +1,193 @@
+/*******************************************************************************
```

Comment:
@bargenilesh  do you have unit test for this class? just want to make sure it won't decrease code coverage.